### PR TITLE
Fix non-avatar agents appearing as avatars

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

**Comment by Atlante45** [comment]
## Test Plan:
- TBD

[ticket](https://highfidelity.manuscript.com/f/cases/18741/Agent-scripts-can-no-longer-be-invisible)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13736/HighFidelity-Beta-PR14170-8e625e9.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13736/HighFidelity-Beta-PR14170-8e625e9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13736/HighFidelity-Beta-Interface-PR14170-8e625e9.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13736/HighFidelity-Beta-PR14170-8e625e9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13736/HighFidelity-Beta-Interface-PR14170-8e625e9.exe

**Comment by qligmaloney** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13948/HighFidelity-Beta-PR14170-8e625e9.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13948/HighFidelity-Beta-PR14170-8e625e9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13948/HighFidelity-Beta-Interface-PR14170-8e625e9.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13948/HighFidelity-Beta-PR14170-8e625e9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13948/HighFidelity-Beta-Interface-PR14170-8e625e9.dmg

File: assignment-client/src/avatars/AvatarMixerSlave.cpp [hunk 1]
```diff
@@ -380,6 +380,11 @@ void AvatarMixerSlave::broadcastAvatarDataToAgent(const SharedNodePointer& node)
             if (lastSeqToReceiver == lastSeqFromSender && lastSeqToReceiver != 0) {
                 ++numAvatarsHeldBack;
                 shouldIgnore = true;
+            } else if (lastSeqFromSender == 0) {
+                // We have have not yet recieved any data about this avatar. Ignore it for now
```
Comment (SimonWalton-HiFi) [inline assignment-client/src/avatars/AvatarMixerSlave.cpp]: Typo - should be "received"

Comment (Atlante45) [reply assignment-client/src/avatars/AvatarMixerSlave.cpp]: I hate this word, I hate this word, I hate this word!

**Pull request merged**